app-backend/migrations/src_migrations/main/scala/162.scala

```diff
@@ -0,0 +1,41 @@
+import slick.jdbc.PostgresProfile.api._
+import com.liyaos.forklift.slick.SqlMigration
+
+object M162 {
+  RFMigrations.migrations = RFMigrations.migrations :+ SqlMigration(162)(
+    List(
+      sqlu"""
+    ALTER TABLE annotations
+      ADD COLUMN project_layer_id UUID,
+      ADD CONSTRAINT annotations_project_layer_id_fkey
+      FOREIGN KEY (project_layer_id)
+      REFERENCES project_layers(id)
+      ON DELETE CASCADE;
```

Comment:
I don't think you need to define the constraint separately from the column like this
```sql
ALTER TABLE annotations
ADD COLUMN project_layer_id UUID REFERENCES project_layers(id) ON DELETE CASCADE;
```